src/saving/SaveList.tscn

```diff
@@ -43,10 +44,17 @@ margin_bottom = 715.0
 size_flags_horizontal = 3
 size_flags_vertical = 3
 
+[node name="NoSavesLabel" type="Label" parent="MarginContainer/VBoxContainer"]
+visible = false
+margin_right = 1270.0
+margin_bottom = 25.0
+text = "NO_SAVEGAMES_FOUND"
+align = 1
```

Comment:
Having the text vertically aligned to the center of the save list might be better.